File: python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, name, document, email, password, passenger, typeCarAccepted, seatsMaterial):
        super().__init__(name, document, email, password, passenger)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

# test que el super objeto Car funciona
#uber = UberVan("Angel peraza", "1033767", "angel@gmail.com", "4561223*", 5, ["mazda cxr"], ["leather"])
#print(vars(uber))

File: java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {
    Map<String, ArrayList<String>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    private Integer passenger;

    public UberVan(String license, Account driver){
        super(license, driver);
    }

    public UberVan(String license, Account driver, 
    Map<String, ArrayList<String>> typeCarAccepted,
    ArrayList<String> seatsMaterial){
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    public Integer getPassenger(){
        return passenger;
    }

    @Override
    public void setPassenger(Integer passenger) {
        if(passenger == 6){
            this.passenger = passenger;
        }else{
            System.out.println("Necesitas asignar 6 pasajeros");
        }
    }

    
}